### PR TITLE
Use yapf to format all python files and keep them formatted

### PR DESCRIPTION
**Pull request by eventualbuddha**

### CODE REVIEW

File: arlo_server/__init__.py [hunk 1]
```diff
@@ -26,51 +27,59 @@ def create_election(election_id=None):
...
+                                     for choice in contest.choices])
 
-        round_contests = RoundContest.query.filter_by(contest_id = contest.id).order_by('round_id').all()
+        round_contests = RoundContest.query.filter_by(
+            contest_id=contest.id).order_by('round_id').all()
         for round_contest in round_contests:
             for result in round_contest.results:
-                contests[contest.id][result.targeted_contest_choice_id] += result.result
+                contests[contest.id][
+                    result.targeted_contest_choice_id] += result.result
```
Comment (umbernhard) [inline arlo_server/__init__.py]: Any idea if there's some way to get it to not break up brackets like this? I'm not seeing anything specific in PEP8 about it, is it pythonic?

File: audits/bravo.py [hunk 2]
```diff
@@ -233,19 +238,20 @@ def get_sample_sizes(self, contests, margins, sample_results):
...
+            s_w = 0
             p_l = 0
             best_loser = ''
             worse_winner = ''
 
-
             # For multi-winner, do nothing
-            if 'numWinners' not in contests[contest] or contests[contest]['numWinners'] != 1:
+            if 'numWinners' not in contests[
+                    contest] or contests[contest]['numWinners'] != 1:
```
Comment (umbernhard) [inline audits/bravo.py]: Here's another instance of it splitting on braces, it kind of makes the code hard to read.

File: audits/bravo.py [hunk 3]
```diff
@@ -340,19 +343,20 @@ def get_test_statistics(self, margins, sample_results):
...
             for winner in winners:
-                T[(winner,)] = 1
+                T[(winner, )] = 1
 
         for cand, votes in sample_results.items():
             if cand in winners:
                 for loser in losers:
-                    T[(cand, loser)] *= (winners[cand]['swl'][loser]/0.5)**votes
+                    T[(cand,
+                       loser)] *= (winners[cand]['swl'][loser] / 0.5)**votes
```
Comment (umbernhard) [inline audits/bravo.py]: And again (I'm not gonna beat a dead horse, just wanted to provide some pointers to places where I'm not sure yapf is making the code more readable)

**Comment by eventualbuddha** [comment]
@umbernhard okay, I upped the allowed columns per line to 100. PTAL?

**Comment by umbernhard** [comment]
@eventualbuddha much better!

**Pull request merged**